js/src/tooltip.js

```diff
@@ -438,6 +439,15 @@ const Tooltip = (($) => {
       return AttachmentMap[placement.toUpperCase()]
     }
 
+    _cleanTipClass() {
```

Comment:
Using the `CLASS_PREFIX` is clever !
You can make this code shorter and more efficient thought:
```js
// On top of the file
const TETHER_PREFIX_REGEX = new RegExp(`(\S*)${CLASS_PREFIX}(\S*)`, 'g');

const $tip = $(this.getTipElement())
$tip.removeClass($tip.attr('class').match(TETHER_PREFIX_REGEX);
```
That makes the code more readable and  would avoid the multiple calls to Jquery to remove the classes one by one. It also avoid the loop and the assigning and reassigning of temporary variables multiple time.